docs/wip_glassbox/form-extension.md

```diff
@@ -0,0 +1,107 @@
+# Form Extension
+In this document we will be explaining the actual state of our forms and their extensions. Right now, 
+solution of extending forms is not complete and there will be several tasks that will assure better extending.
+
+At this time we do not have every single form in our application ready for extension, the list of already 
```

Comment:
well not really because OrderFormType is not ready too. And actually, shouldnt this be used same way in front formtypes?

Reply:
I mean if front FormTypes shouldnt be done same way.

Reply:
Ah I forgot about OrderFormType. 

No, frontend forms will not have such changes because almost every frontend form does look like differently and tho it cannot be universal.

It would be nice if you could update it so it is easier to understand current status of reworking forms.

Reply:
I dont understand what you want me to update.

Reply:
That only those two forms need to be updated.

Reply:
Okay, I wonder if there is US for that OrderFormType. Dont you know if it exists or i should create it?